react/features/base/media/middleware.js

```diff
@@ -77,14 +80,23 @@ function _setRoom({ dispatch, getState }, next, action) {
     typeof videoMuted === 'undefined'
         && (videoMuted = config.startWithVideoMuted);
 
+    audioMuted = Boolean(audioMuted);
+    videoMuted = Boolean(videoMuted);
+
     // Apply the config.
 
+    sendEvent(`startmuted.audio.${audioMuted ? 'enabled' : 'disabled'}`);
+    sendEvent(`startmuted.video.${videoMuted ? 'enabled' : 'disabled'}`);
```

Comment:
Don't we want to distinguish those events from the same send on JitsiConferenceEvents.STARTED_MUTED?

Reply:
Hmm. I don't know. The functionality is the same. Do we?